helm/cluster-aws/values.schema.json

```
@@ -205,6 +205,12 @@
                         ]
                     }
                 },
+                "createCiliumNetworkPolicies": {
+                    "type": "boolean",
+                    "title": "Create CiliumNetworkPolicy manifests for DNS and proxy access in critical namespaces",
```

This title is a bit long. If needed, information can be moved into the description.

```suggestion
                    "title": "Create CiliumNetworkPolicy manifests",
```